Training/Model/Source/CustomSelect.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Codifi_Training
+ *
+ * @copyright   Copyright (c) 2021 Codifi
+ * @author      Pavel Zelenevich <pzelenevich@codifi.me>
+ */
+
+namespace Codifi\Training\Model\Source;
+
+/**
+ * Class CustomSelect
+ * @package Codifi\Training\Model\Source
+ */
+class CustomSelect extends \Magento\Eav\Model\Entity\Attribute\Source\AbstractSource
+{
+    /**
+     * @return array|array[]|null
+     */
+    public function getAllOptions()
+    {
+        if (null === $this->_options) {
+            $this->_options = [
+                ['label' => __('Yes'), 'value' => 1],
+                ['label' => __('No'), 'value' => 0],
+            ];
+        }
+        return $this->_options;
+    }
+
+    /**
+     * @param int|string $value
```

Comment:
I suppose there is no need to rewrite this method (see implementation in vendor/magento/module-eav/Model/Entity/Attribute/Source/AbstractSource.php:65)